doc/camera_and_viewport.md

```diff
@@ -29,17 +29,17 @@ your needs):
 
 When using `BaseGame`, the operations performed by the viewport are done automatically to every
 render operation, and the `size` property in the game, instead of the logical widget size, becomes
-the size as seen through the viewport. If for some reason you need to access the original real
-logical pixel size, you can use `canvasSize`. For a more in depth description on what each Viewport
-does and how it operates, check the documentation on its class.
+the size as seen through the viewport together with the zoom of the camera. If for some reason you
+need to access the original real logical pixel size, you can use `canvasSize`. For a more in depth
+description on what each Viewport does and how it operates, check the documentation on its class.
 
 ## Camera
 
 Unlike the Viewport, the Camera is a more dynamic Canvas transformation that is normally dependent
```

Comment:
```suggestion
Unlike the `Viewport`, the `Camera` is a more dynamic `Canvas` transformation that is normally dependent
```

Not part of your PR, but I feel this sentence is not correctly worded? "is a more ... that is normally dependent on" and then a list?